### PR TITLE
Track sign in redirect error

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Track sign in redirect error

[stage-2]

## Motivation and Context
Better tracking of errors.

## How Has This Been Tested?
Tested locally and on staging.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: src/scripts/components/userstate/services/svc-openid-connect.js [hunk 1]
```diff
@@ -148,6 +148,10 @@
             }).catch(function (err) {
               $log.error('Authentication Error from Redirect: ', err);
 
+              openidTracker('sign in redirect failed', {}, {
+                errorMessage: err.message
```
Comment (ezequielc) [inline src/scripts/components/userstate/services/svc-openid-connect.js:152]: @alex-deaconu Will the `.message` field be always available? I am wondering if we should at least add a check around it (`err && err.message`), to make sure we will log the event. Thanks!

**Review by ezequielc — APPROVED** [review]

File: src/scripts/components/userstate/services/svc-openid-connect.js (hunk 1 above)
Comment (alex-deaconu) [reply src/scripts/components/userstate/services/svc-openid-connect.js]: @ezequielc good call, updated. Thanks.

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Thanks for the update 👍

**Pull request merged**